src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -2370,7 +2370,18 @@ export class PostgresStorageAdapter implements StorageAdapter {
       }
     }
 
-    const qs = `SELECT ${columns.join()} FROM $1:name ${wherePattern} ${sortPattern} ${limitPattern} ${skipPattern} ${groupPattern}`;
+    if (groupPattern) {
+      columns.forEach((e, i, a) => {
+        if (e && e.trim() === '*') {
+          a[i] = '';
+        }
+      });
+    }
+
+    const qs = `SELECT ${columns
+      .filter(Boolean)
+      .join()} FROM $1:name ${wherePattern} ${skipPattern} ${groupPattern} ${sortPattern} ${limitPattern}`;
```

Comment:
Can you explain the problem you are trying to fix? Just so we can determine if there are side-effects.

Reply:
@dplewis 
The queries used to fail when * was used with GROUPBY operators and the order of few operators where off when GROUPBY was present.

Reply:
@dplewis 
Can you please merge this? All the existing test cases that was there to handle groupBy and other conditions are working as it sits.

Reply:
@dplewis Anything on this?